frontend/src/pages/TracesExplorer/Filter/filterUtils.ts

```diff
@@ -8,10 +9,14 @@ import {
 import { TagFilterItem } from 'types/api/queryBuilder/queryBuilderData';
 import { DataSource } from 'types/common/queryBuilder';
 
-export const AllTraceFilterKeyValue = {
+const RESOURCE_DEPLOYMENT_ENVIRONMENT = convertMetricKeyToTrace(
+	'resource_deployment_environment',
+);
```

Comment:
You don't need to take `resource_deployment_environment` and convert it. You can directly use `deployment.environment`.  Conversion is only required when we go from metrics to traces/logs.

Reply:
Done

Reply:
Done, @srikanthccv